pyproject.toml

```diff
@@ -19,3 +19,6 @@ classifiers = [
     "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3",
 ]
+
+[tool.isort]
+profile = "black"
```

Comment:
You also need to set the multiline output line mode.

https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

Reply:
According to the isort docs here: https://pycqa.github.io/isort/docs/configuration/profiles.html

That is `multi_line_output = 3` is included in the black profile. Did you mean something else?

Reply:
Confirmed with the isort author that adding `multi_line_output` is unnecessary with the profile:

https://github.com/PyCQA/isort/pull/1858

Reply:
Neato! Thanks for doing this. I was just looking to make the same PR. :)